addons/website/static/src/snippets/s_website_form/001.scss

```diff
@@ -83,6 +83,11 @@
             background-color: darken($o-gray-200, 4.5%);
         }
     }
+
+    // Signup form container width
+    &.o_signup > div.container {
+        max-width: 500px !important;
+    }
```

Comment:
Why this change here in snippet related scss file, signup form in not related to snippet